server/ai_live_video.go

```diff
@@ -199,30 +198,47 @@ func startEventsSubscribe(ctx context.Context, url *url.URL, params aiRequestPar
 				continue
 			}
 
-			stream := params.liveParams.stream
-
-			if stream == "" {
-				clog.Infof(ctx, "Stream ID is missing")
-				continue
-			}
-
-			var status monitor.PipelineStatus
+			var status map[string]interface{}
```

Comment:
FTR: These events are not status anymore. They are just events. So IMO you should rename this var to just `event` and only treat it as a `status` when you check `if typeVal, ok := status["type"]; ok && typeVal == "status" {`

This is just naming, but we will avoid more conceptual confusion in the future as we iterate on this.